movie-showcase/src/routes/display/+page.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+    import Carousel from "svelte-carousel";
+    import { movieArray } from "../../stores";
+    import { slideDuration } from "../../stores";
+    import { carouselItemCount } from "../../stores";
+</script>
+
+{#if $movieArray.length >= 1}
+    <Carousel
+        particlesToShow={$carouselItemCount}
+        autoplay
+        autoplayDuration={0}
+        duration={$slideDuration * 1000}
+        timingFunction="linear"
+        dots={false}
+        arrows={false}
+        swiping={false}
+    >
+        {#each $movieArray as item}
+            <div class="card">
+                <div class="poster">
+                    <img alt="poster" src={item.Poster} />
+                </div>
+                <div class="movie-description">
+                    <div class="title">
+                        {item.Title}
+                    </div>
+                    <div class="year">
+                        {item.Year} -
```

Comment:
```suggestion
                        {item.Year}
```

Is the dash intentional after the year ?

Reply:
Yes it's intentional, thought it looked better. Should I remove it?